Foundation/NSOrderedSet.swift

```diff
@@ -243,25 +232,61 @@ extension NSOrderedSet {
     // to the original ordered set will "show through" the facade and it will
     // appear to change spontaneously, since a copy of the ordered set is not
     // being made.
-    public var array: [AnyObject] { NSUnimplemented() }
-    public var set: Set<NSObject> { NSUnimplemented() }
-    
-    public func enumerateObjectsUsingBlock(_ block: (AnyObject, Int, UnsafeMutablePointer<ObjCBool>) -> Void) { NSUnimplemented() }
-    public func enumerateObjectsWithOptions(_ opts: NSEnumerationOptions, usingBlock block: (AnyObject, Int, UnsafeMutablePointer<ObjCBool>) -> Void) { NSUnimplemented() }
-    public func enumerateObjectsAtIndexes(_ s: NSIndexSet, options opts: NSEnumerationOptions, usingBlock block: (AnyObject, Int, UnsafeMutablePointer<ObjCBool>) -> Void) { NSUnimplemented() }
+    public var array: [AnyObject] {
+        return _orderedStorage.bridge()
```

Comment:
This is probably good enough for now, because it's busted on Darwin too - but note that the comment here describes this class as returning a proxy object for this method and also `set`. The bridge does a copy, so no proxying is happening.